api/web/src/state.js

```diff
@@ -1,3 +1,22 @@
 export const STARTED = "STARTED";
 export const STOPPED = "STOPPED";
-export const IN_PROGRESS = "IN_PROGRESS";
\ No newline at end of file
+export const IN_PROGRESS = "IN_PROGRESS";
+
+
+export const computeState = (workloads) =>
+{
+    const workloadInProgress = workloads.find(w => !computeIsFunction(w) && w.current === 0 && w.target > 0 && !computeIsDailyClean(w))
+    if(workloadInProgress) return IN_PROGRESS;
+
+    const workloadStopped = workloads.find(w => !computeIsFunction(w) && w.target === 0 && !computeIsDailyClean(w))
+    return workloadStopped != null ? STOPPED : STARTED;
+}
+
+
+export const computeIsFunction = (workload) => {
+    return (workload && workload.labels && workload.labels["axa.com/function"] === "true");
+}
+
+export const computeIsDailyClean = (workload) => {
+    return (workload && workload.labels && workload.labels["axa.com/dailyclean"] === "false");
```

Comment:
Attention ici, je pense qu'il y a un problème dans le nom de la méthode, j'imagine que tu voulais dire computeIsNotDailyclean, vu que le label est à false ;) Autre point d'attention, malheureusement, la règle de gestion est inversée sur les StatefulSets ... Il faudrait que l'on demande côté OPS pourquoi nous avons fait comme ça dans le script... Du coup, pour ne pas dupliquer la règle de gestion, le temps de valider tout ça et d'éventuellement en changer le comportement, je te conseille de plutôt utiliser workload.isDailycleaned ;) Ca fonctionnera direct quelque soit le type de workload :)

Reply:
Tiens voici l'illustration de ce que ça implique : 
![image](https://github.com/AxaFrance/dailyclean/assets/71655527/73509f0a-682e-4baf-99fb-331e30d49e54)

Reply:
C'est information devrais être dans la doc du readme :p

Reply:
J'ai retiré le label de mysql, sauf que par défaut, si pas de label sur un StatefulSets, il est ignoré par Dailyclean. Il faudrait que l'on revienne peut être là dessus, mais en attendant, je te réitère mon conseille, utilise direct le booléen que je renvoie ;) Il fait ce distinctif de comportement entre Deployments et StatefulSets ;)

Reply:
Oui c'est fait :)

Reply:
> 

Oui, tu as parfaitement raison ;=)